lib/elastic_record/index/manage.rb

```diff
@@ -76,7 +76,7 @@ def aliased_names
       end
 
       def all_names
-        connection.json_get("/#{alias_name}/_mapping/#{model.doctype.name}/").keys
+        connection.json_get("/#{alias_name}/_mapping").keys
```

Comment:
This change doesn't really have anything to do with the ES6 upgrade. I found that the endpoint actually wouldn't return **all** indexes that the given alias represents, only the first one. Removing the doctype seemed to fix this.